README.mdown

```diff
@@ -95,11 +95,16 @@ should be added like this to `yas-snippet-dirs`:
 
 # Manual, issues etc
 
-Please refer to the comprehensive [documentation][docs] for full
-customisation and support.  If you find a bug in the code or in the
-documentation, please report it to the main Emacs bug list,
-bug-gnu-emacs@gnu.org, and put "yasnippet" somewhere in the subject.
-Alternatively, you may use the [Github issue tracker][issues].
+There's comprehensive [documentation][docs] on using and customising
+YASnippet.
+
+There's a [list of support issues][support-issues], with solutions to
+common problems and practical snippet examples.
+
+The [Github issue tracker][issues] is where most YASnippet-related
+discussion happens.  Nevertheless, since YASnippet is a part of Emacs,
+you may alternatively report bugs to the main Emacs bug list,
+bug-gnu-emacs@gnu.org, puting "yasnippet" somewhere in the subject.
```

Comment:
s/puting/putting/. Otherwise looks good to me.

Reply:
dang